SConstruct

```diff
@@ -33,8 +33,8 @@ def get_version():
     with open("VERSION", "r") as version_file:
         return version_file.read().rstrip()
 
-API_VERSION_MAJOR = "1"
-API_VERSION_MINOR = "6"
+API_VERSION_MAJOR = "2"
+API_VERSION_MINOR = "0"
```

Comment:
Yeah.  This is really what needs to be done.  Just the fact that you had to rebuild a client (tool, middleware, etc.) to get it to work with these changes is exactly the signal that the major version needs bumping.